### PR TITLE
PXP-639: [CLI] Ctrl-e stopped working on the latest codebase

### PR DESCRIPTION
**Pull request by jk-gan**
## Summary

Ticket: [PXP-639](https://mindvalley.atlassian.net/browse/PXP-639)

### Changed
- [x] Add an ID to determine whether the API response is outdated or not
For example, the `log_entries` type is `(String, Vec<LogEntry>)`, the first value is the ID. Every time the state is changed (Namespace, version, time filter, log severity and etc), a new ID will be generated. And if the API ID is not the same with the latest ID, it won't append to the `Vec`. This prevent the existing outdated API call results pollute the log vector.

[PXP-639]: https://mindvalley.atlassian.net/browse/PXP-639?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/175#" title="Author self-approved">jk-gan</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by onimsha — APPROVED** [review]

**Pull request merged**
